Util/scripts/write_probdata.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python3
+
+"""
+This routine parses plain-text parameter files that list runtime
+parameters for use in our codes.  The general format of a parameter
+is:
+
+max_step                            integer            1
+small_dt                            real               1.d-10
+xlo_boundary_type                   character          ""
+octant                              logical            .false.
+
+This specifies the runtime parameter name, datatype, and default
+value.
+
+An optional 4th column can be used to indicate the priority -- if,
```

Comment:
It says here the 4th column should be a number, but in the parameter files for Sedov and flame_wave it’s y/n?

Reply:
oh right, that comment is wrong.  I'll fix that.  The 4th column indicates whether the variable should be in the namelist